src/sentry/grouping/enhancer/enhancement-configs/mobile@2021-04-02.txt

```diff
@@ -0,0 +1,333 @@
+## * The default configuration of stacktrace grouping enhancers
```

Comment:
I don't think this file is supposed to be new

Reply:
Good catch, the file existed in two places with identical content. I removed the unused one.